yarn.lock

```diff
@@ -30,144 +30,157 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
-"@commitlint/cli@8.3.5", "@commitlint/cli@^8.3.5":
-  version "8.3.5"
-  resolved "https://registry.yarnpkg.com/@commitlint/cli/-/cli-8.3.5.tgz#6d93a3a8b2437fa978999d3f6a336bcc70be3fd3"
-  integrity sha512-6+L0vbw55UEdht71pgWOE55SRgb+8OHcEwGDB234VlIBFGK9P2QOBU7MHiYJ5cjdjCQ0rReNrGjOHmJ99jwf0w==
-  dependencies:
-    "@commitlint/format" "^8.3.4"
-    "@commitlint/lint" "^8.3.5"
-    "@commitlint/load" "^8.3.5"
-    "@commitlint/read" "^8.3.4"
-    babel-polyfill "6.26.0"
-    chalk "2.4.2"
+"@babel/runtime@^7.9.6":
+  version "7.11.0"
+  resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.11.0.tgz#f10245877042a815e07f7e693faff0ae9d3a2aac"
+  integrity sha512-qArkXsjJq7H+T86WrIFV0Fnu/tNOkZ4cgXmjkzAu3b/58D5mFIO8JH/y77t7C9q0OdDRdh9s7Ue5GasYssxtXw==
+  dependencies:
+    regenerator-runtime "^0.13.4"
+
+"@commitlint/cli@9.1.1", "@commitlint/cli@^9.1.1":
+  version "9.1.1"
+  resolved "https://registry.yarnpkg.com/@commitlint/cli/-/cli-9.1.1.tgz#24277e6c1e616191a34a7182a1d3e73369a34f98"
+  integrity sha512-ye9vUZz8Niv77e5ecBWnA2PnuPF+vYy/3BfSZx8sEaNjozJWUolTk/fuqTsoij3bseZBVObd+cVqZgGk8lfEdw==
+  dependencies:
+    "@babel/runtime" "^7.9.6"
+    "@commitlint/format" "^9.1.1"
+    "@commitlint/lint" "^9.1.1"
+    "@commitlint/load" "^9.1.1"
+    "@commitlint/read" "^9.1.1"
+    chalk "4.1.0"
+    core-js "^3.6.1"
     get-stdin "7.0.0"
-    lodash "4.17.15"
+    lodash "^4.17.15"
```

Comment:
Enables lodash upgrade to v4.17.19

Reply:
@arnaudbesnier i don't think that it's allowed in the `yarn.lock` file, because the purpose of this file is actually to fix the versions and share the same versions across different environments.

You need to:
1. Upgrade lodash to 4.17.19 using yarn
2. Commit the generated yarn.lock file

If this syntax is not allowed (which I did not verify yet), when asked ton install, yarn will simply re-generate the file and I'm not sure that this change will have any effect, if it has, yarn will upgrade more than just this dependency.

Reply:
Is it enough to upgrade lodash on an existing install?

Reply:
On the Rails agent, `package.json` is only used for development tools (ie `devDependencies`).
The Rails agent dependencies are the Ruby `gems` you'll find in the `Gemfile` (not the npm dependencies in the `package.json`)